index.js

```diff
@@ -4,6 +4,8 @@ const cookieParser = require('cookie-parser')
 const expressSession = require('express-session')
 const MemoryStore = require('session-memory-store')(expressSession)
 
+require('dotenv').config()
```

Comment:
not sure if we really should use it... do you have some hoster/deployment scenario which honors this way to pass env?